Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -1653,6 +1663,144 @@ namespace AzToolsFramework
             return true;
         }
 
+        void PrefabPublicHandler::AddNewEntityToSortOrder(
```

Comment:
This is a decidedly inelegant solution to the duplicate entity ordering issue right now, but it might serve as a reference point for where we might want to build helper APIs and safeguards for moving towards a more DOM-authoritative model (the sort order definitely should be 100% owned by the prefab DOM long term, there's no reason it should need to be propagated to a physical component as it's purely authoring data).